src/message/__tests__/messagesReducer-test.js

```diff
@@ -117,7 +129,7 @@ describe('messagesReducer', () => {
       });
       const action = deepFreeze({
         type: EVENT_UPDATE_MESSAGE,
-        messageId: 3,
+        messageId: [3],
```

Comment:
Hmm, it's surprising this one makes it past the type-checker! Would be good to identify why.

(I think this change is unintentional and can be simply dropped.)

Reply:
Thanks for catching this!

> Hmm, it's surprising this one makes it past the type-checker!

Alas, this file isn't type-checked at all.

Reply:
Ah, of course :-/ Looks like that's still true of a little over half our test files:
```
$ git ls src/**/__tests__/*.js | wc -l
92

$ git grep -l '@flow' src/**/__tests__/*.js | wc -l
39